pyflakes/checker.py

```diff
@@ -1699,7 +1699,34 @@ def CALL(self, node):
                     with self._enter_annotation():
                         self.handleNode(keyword.value, node)
 
-        self.handleChildren(node)
+        if any(_is_typing(node.func, nt, self.scopeStack)
+                for nt in ("TypedDict", "NamedTuple")):
+            with self._enter_annotation(AnnotationState.NONE):
+                self.handleChildren(node)
```

Comment:
this causes the things to be processed more than once which might be problematic -- though I can't think of a case off the top of my head where this produces incorrect results

Reply:
It is in the same vein as the special case TypeVar